### PR TITLE
Base setup

### PR DESCRIPTION
**Pull request by AvdLee**
First of all, sorry for making this PR so big. Due to Play Day I had build everything at once but a bit too hacky. Therefore, I decided to rewrite it before opening the PR.

It does result in code that looks as good as I think it could. I'm really interested to hear your feedback on it.

### CODE REVIEW

File: Tests/ChangelogProducerTests/IssueResolverTests.swift [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        }
+    }
+    
+    /// It should not extract anything if no issue number is found.
+    func testResolvingNoReferencedIssue() {
+        let description = examplePullRequestDescriptionUsing(closingKeyword: "fixes", issueNumber: nil)
+        XCTAssertTrue(description.resolvingIssues().isEmpty)
+    }
+    
+    /// It should not extract anything if no closing keywork is found.
```
Comment (kairadiagne) [inline Tests/ChangelogProducerTests/IssueResolverTests.swift]:
```suggestion
    /// It should not extract anything if no closing keyword is found.
```

File: Tests/ChangelogProducerTests/XCTestManifests.swift [hunk 2]
```diff
@@ -3,7 +3,7 @@ import XCTest
 #if !canImport(ObjectiveC)
 public func allTests() -> [XCTestCaseEntry] {
     return [
-        testCase(ChangelogProducerTests.allTests),
+        // testCase(ChangelogProducerTests.allTests),
```
Comment (kairadiagne) [inline Tests/ChangelogProducerTests/XCTestManifests.swift]: Why is this code commented out? Do we need it for running the test?

Comment (AvdLee) [reply Tests/ChangelogProducerTests/XCTestManifests.swift]: We're not keeping the SPM tests up to date for now but it's good to keep it in place for if we ever like to add it back. That's why I simply commented it out for now!

File: Sources/ChangelogProducerCore/ChangelogBuilder.swift [hunk 3]
```diff
@@ -0,0 +1,21 @@
+//
+//  ChangelogBuilder.swift
+//  ChangelogProducerCore
+//
+//  Created by Antoine van der Lee on 16/01/2020.
```
Comment (Boris-Em) [inline Sources/ChangelogProducerCore/ChangelogBuilder.swift]: Shouldn't the copyright be included in the files?

Comment (AvdLee) [reply Sources/ChangelogProducerCore/ChangelogBuilder.swift]: Good point, I'll add that!

**Pull request merged**